src/reducers/index.ts

```diff
@@ -18,9 +42,84 @@ function reducer(state: Readonly<StateBase>, action: Action): StateBase {
   };
 }
 
+/**
+ * Exclude these actions from the history completely.
+ */
+export const ACTIONS_EXCLUDED_FROM_HISTORY: ActionType[] = [
```

Comment:
Exactly, the purpose is for history; absent of history functionality that information isn't used, so my recommendation is to keep it close to how/where history is configured. Your call, but thats how it seems to me at the moment.